src/components/QuestionForm/styles.js

```diff
@@ -0,0 +1,23 @@
+import styled from 'styled-components';
+
+export const StyledForm = styled.form`
+  width: 100%;
+  margin: 0px auto;
+  max-width: 32rem;
+`;
+
+export const StyledFormRow = styled.div`
```

Comment:
Would just start calling these `FormRow`, `Form`, `Label` rather than "styled" prefixed. Save some readability, since its clear with the caps